mm/slub.c

```diff
@@ -3991,6 +3991,7 @@ void __check_heap_object(const void *ptr, unsigned long n, struct page *page,
 	struct kmem_cache *s;
```

Comment:
Isn't this function also in slab.c, and is it also broken?

Reply:
Correct, will fix that separately.

Reply:
Done.